### PR TITLE
56 update jupyterlab version and fix visualization in incore lab

### PR DESCRIPTION
**Pull request by Vismayak**

### CODE REVIEW

File: Dockerfile.lab [hunk 1]
```diff
@@ -48,13 +35,11 @@ ENV INCORE=${INCORE} \
 RUN umask 0 && \
   find /opt/conda ! -perm -o=w -print | sort > /tmp/oldfiles && \
   conda config --add channels in-core && \
-  mamba install -c ${CHANNEL} -y \
+  conda install -c ${CHANNEL} -y \
```
Comment (ywkim312) [inline Dockerfile.lab:38]: changed conda in here to mamba and build worked well. Using conda failed to me.

**Comment by ywkim312** [comment]
why the lab have the conda's base env activated automatically?

**Comment by ywkim312** [comment]
I was trying to test incore-docs but I was not able to clone the repository inside it. Maybe because it is running on my vm and it is not probably allowed to connect outside, but not sure. Can you test it in your local or other way to see if you can clone incore-docs repo and test pyincore-viz example notebook in there?

**Comment by ywkim312** [comment]
I tried to run pyincore-viz example notebook in the container but got following error in the very first cell that are importing packages and libraries 
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 4
      1 import os
      2 import geopandas as gpd
----> 4 from pyincore import IncoreClient, FragilityCurveSet, FragilityService
      5 from pyincore.dataservice import DataService
      6 from pyincore.hazardservice import HazardService

File /opt/conda/lib/python3.11/site-packages/pyincore/__init__.py:15
     13 from pyincore.utils.expressioneval import Parser
     14 from pyincore.dataservice import DataService
---> 15 from pyincore.utils.geoutil import GeoUtil
     16 from pyincore.utils.networkutil import NetworkUtil
     17 from pyincore.dataservice import DataService

File /opt/conda/lib/python3.11/site-packages/pyincore/utils/geoutil.py:16
     13 from rtree import index
     14 from shapely.geometry import shape, Point, MultiLineString, LineString
---> 16 import fiona
     17 import uuid
     18 import copy

File /opt/conda/lib/python3.11/site-packages/fiona/__init__.py:41
     37                 if glob.glob(os.path.join(p, "gdal*.dll")):
     38                     os.add_dll_directory(p)
---> 41 from fiona._env import (
     42     calc_gdal_version_num,
     43     get_gdal_release_name,
     44     get_gdal_version_num,
     45     get_gdal_version_tuple,
     46 )
     47 from fiona._env import driver_count
     48 from fiona._show_versions import show_versions

ImportError: libabsl_cord.so.2301.0.0: cannot open shared object file: No such file or directory
```

File: Dockerfile.lab (hunk 1 above)
Comment (robkooper) [reply Dockerfile.lab:38]: any specifics about the error?

**Comment by robkooper** [comment]
For the fiona error, what did you try to run what code?

**Comment by ywkim312** [comment]
> For the fiona error, what did you try to run what code?

I built the image with mamba and tried to run pyincore-viz-exmaple notebook in the built image

File: Dockerfile.lab (hunk 1 above)
Comment (ywkim312) [reply Dockerfile.lab:38]: no it was just the regular build fail when conda was not able to find the package.

Comment (Vismayak) [reply Dockerfile.lab:38]: @robkooper could you also build and let us know if it fails?

**Comment by Vismayak** [comment]
@robkooper using libmamba-solver gave me the same issues with fiona in pyincore-viz as mamba install

File: Dockerfile.lab [hunk 2]
```diff
@@ -15,26 +14,13 @@ RUN apt-get -qq update && apt-get install -y --no-install-recommends \
...
-      #@jupyterlab/geojson-extension
-
 # IN-CORE variables, set these to control versions to be installed
 # use --build-arg CHANNEL=in-core/label/rc to install a RC version
 ARG CHANNEL="in-core"
 ARG PYINCORE="1.12.0"
 ARG PYINCORE_VIZ="1.8.3"
 ARG PYINCORE_DATA="0.6.0"
-ARG INCORE="4.3.0"
+ARG INCORE="4.2.0"
```
Comment (robkooper) [inline Dockerfile.lab:23]: this is the incore version (4.5.0 right now)

**Review by robkooper — CHANGES_REQUESTED** [review]
wrong version for incore

**Comment by ywkim312** [comment]
Built still has a problem. Can you resolve this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,11 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Changed
+- Read Quota from incore-service allocation endpoint [#66](https://github.com/IN-CORE/incore-lab/issues/66)
+- Updated Dockerfiles for hub and lab to latest version
```
Comment (ywkim312) [inline CHANGELOG.md:11]: @Vismayak Instead of latest version, can you specify the actual version number?

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**